pkg/analyzer/lib/src/dart/element/element.dart

```diff
@@ -2392,6 +2392,9 @@ class ElementAnnotationImpl implements ElementAnnotation {
   /// The name of the top-level variable used to mark a class as being sealed.
   static const String _SEALED_VARIABLE_NAME = "sealed";
 
+  /// The name of the top-level variable used to mark a method as unsafe.
```

Comment:
Is this just methods or other members?

Reply:
nit: `Unsafe` is a class, not a variable.

Reply:
Fixed.